src/main.rs

```diff
@@ -1725,6 +2035,18 @@ impl PofToolsGui {
     }
 }
 
+fn closest_approach(line_a: Vec3d, line_b: Vec3d, point: Vec3d) -> Vec3d {
+    let a2p = point - line_a;
+    let a2b = line_b - line_a;
+
+    let a2b_2 = a2b.magnitude() * a2b.magnitude();
```

Comment:
Did you forget you added `magnitude_squared` to pof/src/types.rs?

Reply:
yes :)